charmcraft/env.py

```diff
@@ -19,11 +19,17 @@
 
 import distutils.util
 import os
+import pathlib
 import sys
 
 from charmcraft.cmdbase import CommandError
 
 
+def get_managed_environment_project_path():
+    """Path for project when running in managed environment."""
+    return pathlib.Path("/root/project")
```

Comment:
I wonder if this "constant" shouldn't be provided by the `craft_providers` lib.

Reply:
The existence of this directory is due to application logic, even if consistent with Snapcraft, so I'd rather not introduce it to craft-providers.  However the use of `/root` is an assumption that craft-providers could perhaps break.  It could be moved into charmcraft.providers.CharmcraftBuilddBaseConfiguration as part of the contract, or a runtime call to query the home directory - but doesn't really help much here.

Reply:
I now understood how this is used.

Approving, but consider: maybe this should be a module constant?

```
MANAGED_PROJECT_PATH = pathlib.Path("/root/project")
```

What's the point of a function whose only purpose return that?